changelogs/fragments/67282-remove_options_from_some_vmware_modules_that_aren't_used_in_the_code.yml

```
@@ -0,0 +1,4 @@
+removed_features:
+  - vmware_guest_find - Removed deprecated ``datacenter`` option
+  - vmware_vmkernel - Removed deprecated ``ip_address`` option ; use sub-option ip_address in the network option instead
```

```suggestion
  - vmware_vmkernel - Removed deprecated ``ip_address`` option; use sub-option ``ip_address`` in the ``network`` option instead
```
(same for next line)